platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -106,8 +108,30 @@ org.gradle.buildinit.source.package=my.corp.domain
 
 Names of the generated convention plugins now start with `buildlogic` instead of the package name, making them shorter and cleaner.
 
+#### Generating without interactive questions
+
+There is also a new option `--no-dialog` to disable the interactive dialog entirely.
+It ensures the init command can be completed without any user input.
+This is handy to use in shell scripts to ensure they do not accidentally hang.
```

Comment:
🤔 I left a comment about this below, but I think this is too much the how (no interactive input) vs the what (accept defaults).  

💭 I also wonder if it would be useful to have a build level option to explicitly disallow user input to the build for all tasks. I wouldn't worry about that now.